packages/components/src/internal/components/security/APIWrapper.ts

```diff
@@ -86,6 +96,21 @@ export class ServerSecurityAPIWrapper implements SecurityAPIWrapper {
         });
     };
 
+    deleteContainer = (comment: string): Promise<Record<string, unknown>> => {
+        return new Promise((resolve, reject) => {
+            Security.deleteContainer({
+                comment,
+                success: data => {
+                    resolve(data);
+                },
+                failure: error => {
```

Comment:
Use `handleRequestFailure` here instead.

Reply:
Initially tried this before PR but for Security.deleteContainer I get the TS error `Type 'AjaxHandler' is not assignable to type 'RequestFailure<any>'.`

Reply:
Ah, unfortunate. I should see about getting this to work for AjaxHandler as well or introducing another helper.